pico_w/freertos/ping/lwipopts.h

```diff
@@ -16,6 +16,10 @@
 
 // not necessary, can be done either way
 #define LWIP_TCPIP_CORE_LOCKING_INPUT 1
+
+// ping_thread sets socket receive timeout to 1s
```

Comment:
this comment is a bit confusing since the value is 1 ... the #define is an enable, right?